src/Element/TraversableElement.php

```diff
@@ -157,7 +157,11 @@ public function fillField($locator, $value)
             throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
         }
 
-        $field->setValue($value);
+        if ($field->getTagName() === "select") {
```

Comment:
BrowserKit implementation is more straightforward:

https://github.com/minkphp/MinkBrowserKitDriver/blob/master/src/BrowserKitDriver.php#L414-L417

So is this browserkit driver bug? Should this be fixed in there or..?